etc/bento.env

```diff
@@ -34,7 +34,7 @@ BENTOV2_GATEWAY_INTERNAL_CERTS_DIR=/usr/local/openresty/nginx/certs
 
 # Gateway
 BENTOV2_GATEWAY_IMAGE=ghcr.io/bento-platform/bento_gateway
-BENTOV2_GATEWAY_VERSION=0.13.2
+BENTOV2_GATEWAY_VERSION=pr-21
```

Comment:
We can change to `BENTOV2_GATEWAY_VERSION=edge`, now that the gateway PR has been merged

Reply:
@v-rocheleau done.